tests/python_package_test/test_basic.py

```diff
@@ -8,7 +8,7 @@
 from sklearn.model_selection import train_test_split
 
 import lightgbm as lgb
```

Comment:
Should I override this for the classifier and replace with `shape = [n_samples * n_classes] or shape = [n_samples, n_classes]`?

Reply:
I think it can be `"array-like of shape = [n_samples] or shape = [n_samples * n_classes] (for multi-class task) or shape = [n_samples, n_classes] (for multi-class task) or None, optional (default=None)"`